.idea/dataSources.xml

```diff
@@ -8,5 +8,12 @@
       <jdbc-url>jdbc:postgresql://localhost:5432/HackCamp</jdbc-url>
       <working-dir>$ProjectFileDir$</working-dir>
     </data-source>
+    <data-source source="LOCAL" name="HackCamp@hc24_12" uuid="e4a7f4db-55ea-44f3-a00f-791ec7c97494">
+      <driver-ref>postgresql</driver-ref>
+      <synchronize>true</synchronize>
+      <jdbc-driver>org.postgresql.Driver</jdbc-driver>
+      <jdbc-url>jdbc:postgresql://hc24_12:443/postgres</jdbc-url>
```

Comment:
Wouldn't this only work on your system unless we host it locally exactly how you did?